auth-web/src/util/constants.ts

```diff
@@ -694,13 +695,16 @@ export enum ShortNameResponseStatus {
 export enum EFTRefundType {
     APPROVED = 'APPROVED',
     PENDING_APPROVAL = 'PENDING_APPROVAL',
-    REJECTED = 'REJECTED'
+    DECLINED = 'DECLINED',
+    COMPLETED = 'COMPLETED',
+    ERRORED = 'ERRORED'
 }
 
 export enum EFTRefundTypeDescription {
     APPROVED = 'Approved',
     PENDING_APPROVAL = 'Requested',
-    REJECTED = 'Declined'
+    REJECTED = 'Declined',
```

Comment:
REJECTED needs to be changed to DECLINED, so the refund status type can be properly mapped its description